feat: massive pivot progress

harlan-zw opened the pull request:
WIP
![image](https://github.com/harlan-zw/request-indexing/assets/5326365/3b628be3-e2b0-4a6a-816f-974c3f6563bc)

harlan-zw committed:
progress commit

harlan-zw committed:
fix: improved landing title

harlan-zw committed:
Merge branch 'main' of github.com:harlan-zw/request-indexing into feat/v0.1

harlan-zw committed:
Merge branch 'main' of github.com:harlan-zw/request-indexing into feat/v0.1

harlan-zw committed:
feat: recursive queries

harlan-zw committed:
chore: progress commit

harlan-zw committed:
progress commit

harlan-zw committed:
chore: progress commit

harlan-zw committed:
chore: progress commit

harlan-zw committed:
chore: progress commit

harlan-zw committed:
initial db integration

harlan-zw committed:
fix: improve token generation

harlan-zw committed:
progress commit

harlan-zw committed:
Merge branch 'main' of github.com:harlan-zw/request-indexing into feat/v-1.1

harlan-zw committed:
progress commit

harlan-zw committed:
progress commit

harlan-zw committed:
chore: bump deps

harlan-zw committed:
progress commit

harlan-zw committed:
feat: keyword / pages from db

harlan-zw committed:
feat: daily cron sync

harlan-zw committed:
chore: bump deps

harlan-zw committed:
chore: lint

harlan-zw committed:
feat: improved pages / keyword querying

harlan-zw committed:
chore: misc fixes

harlan-zw committed:
feat: improved pagespeed insights

harlan-zw committed:
google adwords integration

harlan-zw committed:
feat: better local queue support

harlan-zw committed:
feat: new worker architechture and ingest all data

harlan-zw committed:
progress commit

harlan-zw committed:
chore: db

harlan-zw committed:
progress commit

harlan-zw committed:
progress commit

harlan-zw committed:
progress commit

harlan-zw committed:
chore: change license to GPL-3.0

harlan-zw committed:
fix: better landing copy

abdonrd commented:
@harlan-zw have you abandoned the project? It seemed like a very interesting project!

harlan-zw commented:
Hey, thanks for checking in.

I decided to put this on pause for a couple of reasons:
- Finishing off my projects. I have a few outstanding projects that are smaller in scope, I figured I should finish them off as they will actual help me launch this project properly ([Nuxt SEO](https://nuxtseo.com/)).
- The underlying infrastructure needed by the project isn't there just yet. I use NuxtHub for on CloudFlare and it's missing some key features (durable objects, cron), I was spending too much time trying to implement this myself so better to wait. Also, Nuxt UI v3 and Nuxt Content v3 are needed.

I plan to return to this in December, the above with be out and I can go full speed to getting it usable.

abdonrd commented:
Understood! Thanks for the update!

harlan-zw committed:
feat: homepage SEO meta, prerender guides, Pillar 4 verified research

- Add useSeoMeta() to homepage with optimized title/description
- Enable crawlLinks: true for guide page prerendering
- Add prerender: false route rules for dashboard/admin/account
- Create verified Pillar 4 research (04-fix-indexing-problems-verified.md)
- Flag fabricated stats from original research (62% AI, 100% API, fake Reddit quotes)
- Update 2026 SEO strategy with verified data and revised page plan
- Update Q1 checklist with completed items

harlan-zw committed:
chore: sync

harlan-zw committed:
chore: sync

harlan-zw committed:
fix: better menus

harlan-zw committed:
chore: progress

harlan-zw committed:
chore: progress

harlan-zw committed:
Merge branch 'main' of github.com:harlan-zw/request-indexing into feat/v0.1
